### PR TITLE
Added check to 404 video poster

### LINKED ISSUE #3559: PHP Notice in Notfound tpl
Notice: Undefined variable: video_poster in include() (line 21 of /var/www/vagrant/lib/themes/dosomething/paraneue_dosomething/templates/notfound/node--notfound.tpl.php).

### PR DESCRIPTION
**Pull request by itsjoekent**
Verifies there is a poster image for the 404 video before printing it.

Fixes #3559 

@aaronschachter

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/templates/notfound/node--notfound.tpl.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
       <?php print $subtitle ?>
     </div>
     <?php if (isset($video)): ?>
-      <video src=<?php print $video ?> autoplay="" loop="" poster=<?php print $video_poster ?> ></video>
+      <video src=<?php print $video ?> autoplay="" loop="" poster=<?php if(isset($video_poster)){ print $video_poster; } ?> ></video>
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/notfound/node--notfound.tpl.php]: Kinda picky, but does it make sense to only output poster= if the variable is set? Also should the value be in quotes? Same with the Video src?  I don't know how all this new HTML5 stuff works / what is best practice :older_man:

Comment (itsjoekent) [inline lib/themes/dosomething/paraneue_dosomething/templates/notfound/node--notfound.tpl.php]: fixinz

**Comment by itsjoekent** [comment]
@aaronschachter updated the codez

**Comment by aaronschachter** [comment]
:+1:

**Pull request merged**